navigation/libandroid-navigation/build.gradle

```diff
@@ -4,12 +4,19 @@ android {
   compileSdkVersion rootProject.ext.compileSdkVersion
   buildToolsVersion rootProject.ext.buildToolsVersion
 
+  VERSION_NAME = "${VERSION_NAME}"
+
+
   defaultConfig {
     minSdkVersion rootProject.ext.minSdkVersion
     targetSdkVersion rootProject.ext.targetSdkVersion
     versionCode 1
     versionName "0.3.1"
     testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+    buildConfigField('String', 'MAPBOX_NAVIGATION_VERSION_NAME', "\"${VERSION_NAME}\"")
+    buildConfigField "String", "MAPBOX_NAVIGATION_EVENTS_USER_AGENT",
+        String.format("\"mapbox-navigation-android/%s\"", project.VERSION_NAME)
```

Comment:
Heads up that we'll need to be able to differentiate between user agents `mapbox-navigation-android` and `mapbox-navigation-ui-android` when using the drop in navigation UI.